.github/workflows/pr-analysis-gradle.yml

```diff
@@ -35,10 +35,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Determine Submodules
+        id: check-submodules
+        run: |
+          if [ -f .gitmodules ]; then
+            echo "::set-output name=has_submodules::true"
+          else
+            echo "::set-output name=has_submodules::false"
```

Comment:
`set-output` is deprecated and due to be removed https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
I have got rid of the deprecated command.